akka-stream-tests-tck/src/test/scala/akka/stream/tck/Timeouts.scala

```diff
@@ -11,5 +11,7 @@ object Timeouts {
   def publisherShutdownTimeoutMillis: Int = 3000
 
   def defaultTimeoutMillis: Int = 800
+  
+  def defaultNoSignalsTimeoutMillis: Int = 200
```

Comment:
decreased the time for the "no signals passed within 200ms".
This is the RS TCK equivalent of "expectNoMsg", so we want to have it low because it does not matter that much to wait a long time